docs/content/docs/dev/table/sql/jar.md

```diff
@@ -68,22 +70,24 @@ Flink SQL> REMOVE JAR '/path/hello.jar';
 ADD JAR '<path_to_filename>.jar'
 ```
 
-Currently it only supports to add the local jar into the session classloader.
+Currently it supports to add the jar locates in a local or remote [file system]({{< ref "docs/deployment/filesystems/overview" >}}) into the session classloader.
 
-## REMOVE JAR
+## SHOW JARS
 
 ```sql
-REMOVE JAR '<path_to_filename>.jar'
+SHOW JARS
 ```
 
-Currently it only supports to remove the jar that is added by the [`ADD JAR`](#add-jar) statements.
+Show all added jars in the session classloader which are added by [`ADD JAR`](#add-jar) and [`USING JAR`]({{< ref "docs/dev/table/sql/create" >}}#create-function) statements.
```

Comment:
Why `SHOW JARS` also shows jars in `CREATE FUNCTION USING JAR`? The semantics of `SHOW/ADD/REMOVE JARS` seem not consistent to me. From my understanding, `SHOW JARS` should show the jars added to the session, but `CREATE FUNCTION USING JAR` doesn't affect the current session and functions with using jar referenced by queries are bound to the queries not sessions.

Reply:
After reading spark [add jar](https://spark.apache.org/docs/latest/sql-ref-syntax-aux-resource-mgmt-add-jar.html) and [list jar](https://spark.apache.org/docs/latest/sql-ref-syntax-aux-resource-mgmt-list-jar.html) document, I think you are right, the show jars only list the jar which is added by add jar statement, this seems the semantic is consistent.
However, I've operated the spark sql and hive cli, I found that the list jars statement also shows the jars which are added `Create function ... using jar` statement, the document and actual behavior is not consistent. So regarding the document, I think we can follow the spark.